script/build.sh

```diff
@@ -79,4 +79,3 @@ if [ -n "${forceContentBuild}" ] || [ "${buildtype}" != "vagovdev" ]
 then
     yarn build:content $args
 fi
-
```

Comment:
Extra change here

Reply:
I noticed there was an extra blank line at the end of that file so I removed it.